docs/getting-started.md

```diff
@@ -86,6 +86,9 @@ checked-in [deployment yaml](../build/yamls/antrea.yml):
 kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea.yml
 ```
 
+You can use the same `kubectl apply` command to upgrade to a more recent version
```

Comment:
Should we provide a link here for https://github.com/antrea-io/antrea/blob/main/docs/versioning.md#antrea-upgrade-and-supported-version-skew about Antrea's upgrade policy?

Reply:
Yes, I can add the link, but I don't want to create too much confusion. I don't think anyone is actually splitting the YAML to ensure the controller is upgraded first. What do you think?